repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl

```diff
@@ -76,6 +102,64 @@
                                 </div>
                             </section>
                         </div>
+                        <div class="nhsuk-grid-column-one-third">
+                            <div class="nhsuk-card">
+                                <div class="nhsuk-card__content">
+                                    <h3 class="nhsuk-heading-m">Publication Info</h3>
+                                    <div class="heeuk-link-inactive-state-s">
+                                        <strong>Published: </strong> ${document.publicationDate.getTime()?date?string["${datePattern}"]}
+                                    </div>
+                                    <div class="heeuk-link-inactive-state-s"><strong>Updated: </strong> ${document.properties['hippostdpubwf:lastModificationDate'].getTime()?date?string["${datePattern}"]}</div>
+                                    <#if landingPage??>
+                                        <div class="heeuk-link-inactive-state-s">
+                                            <strong>Topics: </strong>
+                                            ${landingPage.publicationType}
+                                            <#if landingPage.publicationProfessions?has_content>
+                                                <#list landingPage.publicationProfessions as profession>
+                                                    - ${profession}
+                                                </#list>
+                                            </#if>
+                                            <#if landingPage.publicationTopics?has_content>
+                                                <#list landingPage.publicationTopics as topic>
+                                                    - ${topic}
+                                                </#list>
+                                            </#if>
+                                        </div>
+                                        <strong>Estimated reading time:</strong> ${landingPage.readTime} min
+                                    </#if>
+                                </div>
+                            </div>
+                            <#if landingPage?? && landingPage.documentVersions?has_content>
+                                <div class="nhsuk-card">
+                                    <div class="nhsuk-card__content">
+                                        <h3 class="nhsuk-heading-m">Alternative versions</h3>
```

Comment:
Could we handle empty `Alternative versions` boxes (when documents aren't available) ?

<img width="329" alt="Screenshot 2022-12-16 at 08 01 04" src="https://user-images.githubusercontent.com/35143542/208053369-0540323b-5c9a-495f-95dd-c055381e3efb.png">